src/components/Form/Form.tsx

```diff
@@ -0,0 +1,66 @@
+import { Form as AntForm, FormProps as AntFormProps, FormInstance } from 'antd'
+import { PropsWithChildren, ReactElement, ReactNode, useMemo } from 'react'
+
+import { Button } from '../Button'
+
+type FormProps<Values> =
+  PropsWithChildren<AntFormProps<Values>>
+  & {
+  columns?: number
+  gap?: number
+  submitButton?: boolean | string | ((form: FormInstance) => ReactNode)
+}
+
+const defaultSubmitButton = (label: ReactNode = 'Submit'): ReactNode => {
+  return (
+    <Button htmlType={Button.HTMLType.Submit}>
+      {label}
+    </Button>
+  )
+}
+
+const renderSubmitButton = (
+  form: FormInstance,
+  submitButtonProp: boolean | string | ((form: FormInstance) => ReactNode)
+): ReactNode => {
+  switch (typeof submitButtonProp) {
+  case 'boolean':
+    return submitButtonProp && defaultSubmitButton()
+  case 'string':
+    return defaultSubmitButton(submitButtonProp)
+  case 'function':
+    return submitButtonProp(form)
+  default:
+  }
+}
+
+export const Form = <Values, >({
+  children,
+  style: styleProp,
+  submitButton: submitButtonProp = true,
+  columns = 2,
+  gap = 24,
```

Comment:
why accepting a style prop **and** a gap? should this default gap be ensured via the style default?

Reply:
I would keep the prop style available in both the form and formItem mainly to override the default "display: grid" behaviour (even if not very likely). 
Additionally, the prop can be useful probably in future implementations for complex components like drawers with forms, modals with forms, etc.